core/src/main/scala/com/rackspace/com/papi/components/checker/Validator.scala

```diff
@@ -98,8 +107,8 @@ class Validator private (private val _name : String, val startStep : Step, val c
 
 
   private val platformMBeanServer = ManagementFactory.getPlatformMBeanServer()
-  private val objectName = new ObjectName("\"com.rackspace.com.papi.components.checker\":type=\"Validator\",scope=\""+
-                                 name+"\",name=\"checker\"")
+  private val objectName = new ObjectName("com.rackspace.com.papi.components.checker:type=Validator,scope="+
+                                 name+",name=checker")
```

Comment:
I assume the " characters are no longer necessary with the updated library, but that the output is the same.